python/testMP.py

```diff
@@ -9,19 +10,21 @@
     parser.add_argument("--port", default=5005, help="OSC server port")
     args = parser.parse_args()
 
-    r = rosary.Rosary(args.ip, args.port)
+    d = dispatcher.Dispatcher()
+    r = rosary.Rosary(args.ip, args.port, d)
 
     # create a bounce effect on all beads
-    r.add_effect('bounce', r.set_registry['All'])
+    #r.add_effect('bounce', r.set_registry['All'])
+    r.add_effect('bounce', 'all')
```

Comment:
Still not sure if I like that I replaced `r.set_registry['All']` with 'all', but again I'm just trying to get through this right now.